### ISSUE TITLE
Caller number display

### ISSUE THREAD

**Issue by vieridipaola** [InnovateAsterisk/Browser-Phone#366]
Hi,

Currently, Browser Phone displays the caller ID name when available in the buddy list. It does not show the number. One would have to open/edit the buddy to see the number.

Can the each buddy in the Buddy List show the number right after the name (or at least show it as a tooltip when hovering over it). It would be nice to also display it when selecting a buddy (right panel).

**Comment by InnovateAsterisk** [comment]
Well, I used to, but I changed that in case users didn't follow the 100, 101, 102 model for username and sip user id.

It is also possible that people have a sip id like _6a249dae-673d-11ed-9022-0242ac120002_. Although this is often accompanied by a DID, and while this system will try to build Buddies based on DIDs, there may be situations that this isn't possible. 

I have already enabled the ability to subscribe to a sip ID different to the buddy DID, so things are getting there, but i'm also working on accommodating these "unguessable" user ids if used as extensions. 

It may become more and more important considering the fact that webrtc by its very nature has to be online, and possibly accessible from the internet. 

I'm going to include this in the enhancement request here: 
https://github.com/InnovateAsterisk/Browser-Phone/issues/365
As thee two will be both address in a planned update to give user choices to the way the buddy list displays and the way the search results are displayed. (same thing really).
